srcs/HandlingString/HandlingString.cpp

```diff
@@ -0,0 +1,325 @@
+#include "HandlingString.hpp"
+
+std::vector<std::string> HandlingString::inputarg_tomap_without_firstword(std::string const &words)
+{
+	std::string					replaced_words = HandlingString::skip_lastsemicoron(words);
+	std::string					word;
+	std::istringstream			splited_words(replaced_words);
+	std::vector<std::string>	ans;
+
+	splited_words >> word;
+	// std::cout << "word is -> " << word << std::endl;
+	while (splited_words >> word)
+	{
+		// std::cout << "word is -> " << word << std::endl;
+		ans.push_back(word);
+	}
+	return (ans);
+}
+
+bool HandlingString::return_matchpattern(std::string true_word, std::string false_word, std::string tgt_word)
+{
+	if (true_word == tgt_word)
+		return (true);
+	else if (false_word == tgt_word)
+		return (false);
+	return (false);
+}
+
+std::string	HandlingString::skipping_first_emptyword(std::string const &word)
+{
+	std::string	tmp_word;
+	size_t		word_length = word.length();
+	size_t		word_nowposl = 0;
+
+	while (word[word_nowposl] == ' ' || word[word_nowposl] == '\t')
+		word_nowposl++;
+	while (word_nowposl != word_length)
+	{
+		tmp_word = tmp_word + word[word_nowposl];
+		word_nowposl++;
+	}
+	return (tmp_word);
+}
+
+std::string HandlingString::skipping_emptyword(std::string const &word)
+{
+	std::istringstream	splited_words(word);
+	std::string			src;
+	std::string			all_src;
+
+	while (splited_words >> src)
+		all_src = all_src + src;
+	return (all_src);
+}
+
+bool	HandlingString::check_under_intmax(std::string const &word)
+{
+	size_t	pos = 0;
+	size_t	sum = 0;
+
+	while (word[pos] != '\0')
+	{
+		if (std::isdigit(word[pos]) == false && word[pos] != ';')
+			return (false);
+		pos++;
+	}
+	if (pos > 12)  // INT_MAXの桁数を明らかに超えるなら計算の必要はない
+		return (false);
+	pos = 0;
+	while (word[pos] != '\0')
+	{
+		sum = sum * 10 + (word[pos] - '0');
```

Comment:
```suggestion
		sum = sum * 10 + (word[pos] - '0');
```
IMO:
`word[pos] - '0'`は`to_digit()`作っても良さそう

Reply:
done